fabric-object-builders-v0/src/main/java/net/fabricmc/fabric/api/block/FabricMaterialBuilder.java

```diff
@@ -34,59 +39,64 @@ public FabricMaterialBuilder(DyeColor color) {
 
 	@Override
 	public FabricMaterialBuilder burnable() {
-		super.burnable();
+		this.delegate.burnable();
 		return this;
 	}
 
 	public FabricMaterialBuilder pistonBehavior(PistonBehavior behavior) {
-		((MaterialBuilderHooks) this).setPistonBehavior(behavior);
+		this.delegate.pistonBehavior(behavior);
 		return this;
 	}
 
 	public FabricMaterialBuilder lightPassesThrough() {
-		((MaterialBuilderHooks) this).invokeLightPassesThrough();
+		this.delegate.lightPassesThrough();
 		return this;
 	}
 
 	@Override
 	public FabricMaterialBuilder destroyedByPiston() {
-		super.destroyedByPiston();
+		this.delegate.destroyedByPiston();
 		return this;
 	}
 
 	@Override
 	public FabricMaterialBuilder blocksPistons() {
-		super.blocksPistons();
+		this.delegate.blocksPistons();
 		return this;
 	}
 
 	@Override
 	public FabricMaterialBuilder allowsMovement() {
-		super.allowsMovement();
+		this.delegate.allowsMovement();
 		return this;
 	}
 
 	@Override
 	public FabricMaterialBuilder liquid() {
-		super.liquid();
+		this.delegate.liquid();
 		return this;
 	}
 
 	@Override
 	public Material.Builder notSolid() {
```

Comment:
This here should be a `FabricMaterialBuilder` (this is the v0 package) but is being handled by #533.

This change is already done in the v1 package